src/main/java/dev/jbang/util/JavaUtil.java

```diff
@@ -81,6 +81,7 @@ private static int determineJavaVersion(Path javaCmd) {
 		String output = Util.runCommand(javaCmd.toString(), "-version");
 		int version = parseJavaOutput(output);
 		if (version == 0) {
+			Util.verboseMsg("Version is 0, reading it from java.version property");
```

Comment:
say something like "Version could not be determined from: '$javaCmd -version', trying 'java.version' property"